### ISSUE TITLE
The 0.2.1 release is not officially recognized by atom.

### ISSUE THREAD

**Issue by TPXP** [georgebrindeiro/scroll-sync#19]
Hello !

It's me, again ! I just noticed that the Atom Package Manager does not correctly recognises my commits as versions, so it still gives the old, non-working `0.1.5` version instead of our shiny `0.2.1`. :cry: 

I probably forgot to tag my commits with the corresponding version (sorry, I was not aware of that feature :confused: ). Could you tag the correct commits on Github ? Just click on the number of releases on the file list, then "Draft a new release", put the correct label and choose the latest corresponding commit (from the list on the right, which enables you to select one of the latest commits).

I'm sorry to bother you again. This time, it should be OK ! :smiley:

**Comment by mark-hahn** [comment]
It's because the updated version hasn't been created yet.  I have to download it, compile it, test it, and then  post it to atom.  It's on my todo list.

Do you want to take over ownership of this package/repo?

**Comment by TPXP** [comment]
Thank you for the update. Although I have intensively tested my changes, I don't think that I have enough experience about atom packages to take ownership of this package so, unless you think you will never have time to try my changes, and since there does not seem to be so much affected users considering the feedback on the bug report, I think we can wait a little bit more to update the package.

Of course, if you could share your testing tips, I would be extremely happy to give them a look and learn ! :smile: 

**TL;DR** : It's your package, so manage it the way think is the best :wink:

**Comment by mark-hahn** [comment]
On Mon, Sep 5, 2016 at 3:02 PM, Thomas P. notifications@github.com wrote:

> Of course, if you could share your testing tips,

Even when I had tons of time for testing, I just used it and tried
everything I could think of.  I should do unit testing but I'm old and
never started doing it.

**Comment by TPXP** [comment]
If that is all you mean by testing, I think that I have done it now. I've been using the module for a few days, did some weird stuff with it to see how stable it was and, so far, everything seems to work properly !

I just noticed a little gap if soft wrap is triggered, especially if the panes do not have the same width. Even though [there is an API for this](https://atom.io/docs/api/search/v1.10.2?utf8=%E2%9C%93&q=soft), I'm sorry I don't feel like hacking this for now. Perhaps open an issue and ask for help ?

Also, if a playful user wants to toggle the module while having a system page opened (settings for example), something goes wrong and a red `undefined` error message appears. It's about adding another `if` in the code, so if you find this important, I can do it and submit another pull request, but I think I've done enough hacking with this module for now, so this can wait a bit! :laughing:

**Comment by CyberPunkCodes** [comment]
It doesn't look like anyone has created a release yet. Atom just installed 0.1.5.

If APM works similar to NPM and packagist, just create a new tag (following same formatting as other tags) based off the master branch. It should update APM in a few minutes automatically.

**Comment by CyberPunkCodes** [comment]
This repo will work again, all you have to do is tag a release! -- I copied the coffee script from the master branch and manually replaced the one in Atom, and it works. So fix the tag, it tags like 30 seconds.
